Makefile

```diff
@@ -435,30 +435,32 @@ uninstall: uninstall_themis uninstall_soter
 # Themis distribution tarball
 #
 
-THEMIS_DIST_FILENAME = $(VERSION).tar.gz
+DIST_DIR = themis-$(VERSION)
+
+# Themis Core source code, tests, docs
+DIST_FILES += docs src tests Makefile VERSION
+DIST_FILES += README.md CHANGELOG.md LICENSE
+DIST_FILES += PKGBUILD.MSYS2 Themis.nsi
+# Supporting files for language wrappers
+DIST_FILES += build.gradle settings.gradle gradle gradlew gradlew.bat
+DIST_FILES += Cargo.toml
+DIST_FILES += Cartfile Cartfile.resolved
```

Comment:
i am not sure that we should include CocoaPods/Carthage files into distributive. Can't see where they are needed

Reply:
Carthage files will be needed for Themis.xcodeproj to become useable.

I’m not sure whether we need to include _any_ language wrappers at all into this tarball. Might as well distribute only the core source code. I’d gladly drop Android’s gradle, for example, to avoid including its JARs that add 5 MB.